test/options/form.test.js

```diff
@@ -0,0 +1,49 @@
+const assert = require('chai').assert;
+const { Builder, By, until } = require('selenium-webdriver');
+const test = require('selenium-webdriver/testing');
+
+test.describe('Options select form', function() {
+  this.timeout(50000);
+
+  const addr = 'http://localhost:3000';
+  let driver;
+
+  test.beforeEach(() => {
+    driver = new Builder()
+      .forBrowser('chrome')
+      .build()
+
+    login(driver);
+
+  });
+
+  test.afterEach(() => {
+    driver.quit();
+  });
+
+  test.it('Loads', () => {
+    driver.findElement(By.id('logout')).getText()
+      .then(text => {
+        assert.equal(text, 'Log Out');
+      });
+  });
+});
+
```

Comment:
@neight-allen I'm logging in to my app before every test. Do you know of a way to mock this so I don't actually need to auth in every time?